dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -62,11 +58,10 @@ maxConPerThd=Maximum open connections per thread
 maxConPerThd.desc=Limits the number of open connections on each thread.
 
 numConPerThd=Number of cached connections per thread
-numConPerThd.desc=Caches the specified number of connections for each thread.
+numConPerThd.desc=Caches the specified number of connections for each thread. This setting can provide a major improvement on large multi-core (8+) machines by reserving the specified number of database connections for each thread. When you set numConnectionsPerThreadLocal to 1 or more, these connections per thread are stored in thread-local storage. When you use numConnectionsPerThreadLocal, consider the number of application threads and the maximum connections for the connection pool. For best performance, if you have n applications threads, set the maximum pool connections to at least n times the value of the numConnectionsPerThreadLocal attribute. Use the same credentials for all connection requests. For example, if you use 20 application threads, set the maximum pool connections to 20 or more. If you set the value of numConnectionPerThreadLocal attribute as 2 and you have 20 application threads, set the maximum pool connection to 40 or more.
```

Comment:
Instead of "can provide a major improvement on"  change "major" to "performance" like this "can provide a performance improvement on" 

After the last sentence add "If setting numConnectionsPerThreadLocal does not improve performance due to application connection usage patterns that do not benefit from using numConnectionsPerThreadLocal, remove the value from configuration or set the value back to 0."

Reply:
Thanks for reviewing @jmstephensgit - I made your first suggestion but the second suggestion exceeds the character limit for the description field. I tried to make room for it by reducing the overall word count. 
To make room, I removed these two sentences, which supplied contextual info, but no unique recommendations:
 
```
 When you set numConnectionsPerThreadLocal to 1 or more, these connections per thread are stored in thread-local storage. When you use numConnectionsPerThreadLocal, consider the number of application threads and the maximum connections for the connection pool.
```

The full current draft is:
```
Caches the specified number of connections for each thread. This setting can provide a performance improvement on large multi-core (8+) machines by reserving the specified number of database connections for each thread. 
For best performance, if you have n applications threads, set the maximum pool connections to at least n times the value of the numConnectionsPerThreadLocal attribute. 
Use the same credentials for all connection requests. For example, if you use 20 application threads, set the maximum pool connections to 20 or more. If you set the value of numConnectionPerThreadLocal attribute as 2 and you have 20 application threads, set the maximum pool connection to 40 or more. 
If setting numConnectionsPerThreadLocal does not improve performance due to application connection usage patterns that do not benefit from using numConnectionsPerThreadLocal, remove the attribute from configuration or set the value to 0.
```

Let me know what you think- thanks!

Reply:
Th full current draft look good, thanks,